cvat/apps/engine/views.py

```diff
@@ -813,12 +811,11 @@ def perform_update(self, serializer):
         if updated_instance.project:
             updated_instance.project.save()
 
-    def perform_create(self, serializer):
-        super().perform_create(
-            serializer,
-            owner=self.request.user,
-            organization=self.request.iam_context['organization']
-        )
+    def perform_create(self, serializer, **kwargs):
+        kwargs.setdefault('owner', self.request.user)
+        kwargs.setdefault('organization', self.request.iam_context['organization'])
```

Comment:
@kirill-sizov, can these parameters be passed via the serializer?

Reply:
I guess yes, it can be done with
```python
owner = serializer.context["request"].user
organization = serializer.context["request"].iam_context["organization"]
```

Reply:
OK, good to know. It doesn't seem to be applicable, though, because each time the set of parameters is different.